.github/quarkus-github-lottery.yml

```diff
@@ -0,0 +1,38 @@
+notifications:
+  createIssues:
+    repository: "quarkusio/quarkus-github-lottery-reports"
+buckets:
+  triage:
+    # TODO: No such label on quarkusio... ? This won't work without a label.
+    label: "triage/needs-triage"
+    delay: PT0S
+    timeout: P3D
+  # TODO: No area labels on quarkusio... ? Notification to maintainers won't work without area labels.
```

Comment:
See:

* https://github.com/quarkusio/quarkus-github-lottery?tab=readme-ov-file#maintenance

Reply:
i adde two areas for now - layout and blog